pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -259,6 +259,12 @@ func (gc *GarbageCollector) attemptToDeleteWorker() bool {
 		}
 		// retry if garbage collection of an object failed.
 		gc.attemptToDelete.AddRateLimited(item)
+	} else if !n.isObserved() {
+		// requeue if item hasn't been observed yet.
+		// otherwise a virtual node for an item added/removed during a watch outage can get orphaned.
```

Comment:
Please don't use orphaned.  That does other things in this area.

Reply:
reworded comment